src/views/ResultsPage/ResultsPage.jsx

```diff
@@ -145,7 +147,11 @@ const ResultsPage = ({ location, tracking }) => {
     }
   };
 
-  const handleStartOver = () => {
```

Comment:
this function does only one thing, why the need for an argument?

Reply:
The `linkType` determines which link was clicked since this function is used for both `Start Over` and `Try a new search` links which is what eventually drives the correct event tracked.